src/pouchDB/api/comments.ts

```diff
@@ -0,0 +1,192 @@
+import {
+  CommentDoc,
+  CreateCommentDoc,
+  DesignDoc,
+  GetCommentListArg,
+  GetList,
+  IdeaDoc,
+  IndexDoc,
+  LikeDoc,
+} from 'pouchDB/types';
+
+export class CommentModel {
+  _db: PouchDB.Database;
+  constructor(db: PouchDB.Database) {
+    this._db = db;
+  }
+
+  // same priciple as idea.ts getIdeaListByRecent
```

Comment:
```suggestion
  // same getIdeaListByRecent as idea.ts getIdeaListByRecent
```

Maybe you can update the code comments in a better way with all sections